### DISCUSSION TITLE
문자 -> 이메일 대체

### DISCUSSION THREAD

**Discussion by im3zero** [MechanIT/Speech-and-object-recognition-for-elderly-people-living-alone#3] [Ideas]
문자 서비스가 해결이 안돼서 이메일로 대체하는 방법은 어떤가요?

**Comment by Jiisou** [comment]
좋은 것 같습니다! 
이메일과 SMS는 보내는 방법이 다르다고 합니다. 이메일은 인터넷을 통한 표준 프로토콜을 사용하여 쉽게 전송할 수 있지만, SMS는 이동통신 네트워크를 통해야 하므로 SMS 게이트웨이를 통해야만 전송 가능하기 때문에 프로토 서비스 구현은 일단 email로 응급 연락을 대체하는 식으로 구현해야 할 것 같습니다.

제 브랜치에 smtp 모듈을 이용해 전자메일을 보내는 소스코드(smtp_email_test.py)를 업로드 해두었습니다.
